src/org/sosy_lab/java_smt/solvers/boolector/BtorOption.java

```diff
@@ -1,3 +1,11 @@
+// This file is part of JavaSMT,
+// JavaSMT is an API wrapper for a collection of SMT solvers.:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2020 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
 package org.sosy_lab.java_smt.solvers.boolector;
 
 /** We keep this synchronized with "btortypes.h". */
```

Comment:
This looks like code that is derived from code in `btortypes.h`? Depending on the license of that file, we might need to keep the original copyright and license notices.